client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-source-site/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@automattic/components';
+import { StepContainer } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const SiteMigrationSource: Step = function ( { navigation } ) {
+	const translate = useTranslate();
+
+	const handleSubmit = () => {
+		navigation.submit?.();
+	};
+
+	const stepContent = (
+		<div>
+			<p>Enter the url of the site you're migrating.</p>
+			<Button primary onClick={ handleSubmit }>
+				{ translate( 'Continue' ) }
+			</Button>
+		</div>
+	);
+
+	return (
+		<>
+			<DocumentHead title="Which site are you migrating?" />
```

Comment:
Translation too.

Reply:
As I mentioned to @andres-blanco earlier and in the PR notes

> Since the content of this step is placeholder-only, we're not doing any translation calls yet.

When can add the translation calls when we finalize that step in #87604.